scripts/sysman/engine.yml

```diff
@@ -117,7 +133,7 @@ params:
       desc: "[in,out] The properties for the specified engine group."
 --- #--------------------------------------------------------------------------
 type: function
-desc: "Get the activity stats for an engine group"
+desc: "Get the activity stats for an engine group. This function also returns the engine activity inside a Virtual Machine (VM), in the presence of HW Virtualization (SRIOV)"
```

Comment:
Recommend moving the added description to the `details:` section.

Reply:
Fixed